lib/repl.js

```diff
@@ -1067,7 +1067,7 @@ REPLServer.prototype.memory = function memory(cmd) {
           self.lines.level.push({
             line: self.lines.length - 1,
             depth: depth,
```

Comment:
@vsemozhetbyt Didn't know about the guideline :p I had a review in my PR for shorthand that's why!